src/reporting/reportingClient.ts

```diff
@@ -268,20 +617,57 @@ export class ReportingClient {
+   * @summary Get GroupProperties
+   * @param {string} imodelId The iModel Id.
+   * @param {string} mappingId The Mapping Id.
+   * @param {string} groupId The Group Id.
+   * @param {string} accessToken OAuth access token with scope &#x60;insights:read&#x60;
+   * @memberof ReportingClient
+   */
   public async getGroupProperties(
     accessToken: AccessToken,
     iModelId: string,
     mappingId: string,
     groupId: string
   ) {
-    return this._mappingsApi.getGroupproperties(
-      iModelId,
-      mappingId,
-      groupId,
-      accessToken
-    );
+    const properties: Array<GroupProperty> = [];
+
+    let response: GroupPropertyCollection;
+    let continuationToken: string | undefined;
+
+    do {
+      response = await this._mappingsApi.getGroupproperties(
```

Comment:
Not following camel case

Reply:
this is from the generated api
we will fix this later when we get rid of the generated client